scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -759,14 +761,15 @@ class Router(formatOps: FormatOps) {
             Policy.NoPolicy
           }
 
-        val handleImplicit =
```

Comment:
can we avoid renaming and restore the original code? can add comment that `using` is also covered, if you'd like